framework/libra-framework/sources/ol_sources/tests/slow_wallet.test.move

```diff
@@ -25,18 +24,21 @@ module ol_framework::test_slow_wallet {
   // and a validator creation sets the users account to slow.
   fun slow_wallet_init(root: signer) {
       let _set = mock::genesis_n_vals(&root, 4);
-      let list = slow_wallet::get_slow_list();
+      print(&@0x666);
+      // let list = slow_wallet::get_slow_list();
```

Comment:
Please no commented out code unless there's a really good (and also commented) reason.